### PR TITLE
Shuffle API for data sources for easier human consumption

### PR DESCRIPTION
**Pull request by conradocloudera**
1. Put the `doc_id` in the path for triggering index and summary computation
2. Sort the paths to be in alphabetical order

### CODE REVIEW

File: llm-service/app/routers/index/data_source/__init__.py [hunk 1]
```diff
@@ -148,24 +169,20 @@ def get_document_summary(self, data_source_id: int, doc_id: str) -> str:
...
     )
     @exceptions.propagates
     def summarize_document(
         self,
         data_source_id: int,
+        doc_id: str,
         request: SummarizeDocumentRequest,
     ) -> str:
+        print("doc_id", doc_id)
+        print("request", request)
```
Comment (mliu-cloudera) [inline llm-service/app/routers/index/data_source/__init__.py]:
```suggestion
```

**Review by jkwatson — APPROVED** [review]
Good changes. Thanks!

**Review by mliu-cloudera — COMMENTED** [review]
👍 Carefully compared the diff in `llm-service/app/routers/index/data_source/__init__.py`, and it indeed looks like a change from `request.document_id` to `doc_ud`, and a pure rearranging for everything else.

File: llm-service/app/routers/index/data_source/__init__.py (hunk 1 above)
Comment (conradocloudera) [reply llm-service/app/routers/index/data_source/__init__.py]: I removed it in between. Good catch!

**Pull request merged**